constants.bal

```diff
@@ -0,0 +1,257 @@
+#              consistent-prefix, monotonic reads, monotonic writes, read-your-writes, and write-follows-reads 
+#              guarantees
+# + EVENTUAL - `Eventual` consistency level where there's no ordering guarantee for reads. In the absence of any further 
+#               writes, the replicas eventually converge.
+public enum ConsistencyLevel {
+    STRONG = "Strong",
+    BOUNDED = "Bounded",
+    SESSION = "Session",
+    EVENTUAL = "Eventual"
+}
+
+# Represents whether to include or exclude the document in indexing.
+#
+# + INCLUDE - `Include` adds the document to the index
+# + EXCLUDE - `Exclude` omits the document from indexing
+public enum IndexingDirective {
+    INCLUDE = "Include",
+    EXCLUDE = "Exclude"
+}
+
+# Represents the type of an Index.
+# 
+# + HASH - `Hash` indexes are useful for equality comparisons
+# + RANGE - `Range` indexes are useful for equality, range comparisons and sorting
+# + SPATIAL - `Spatial` indexes are useful for spatial queries
+public enum IndexType {
+    HASH = "Hash",
+    RANGE = "Range",
+    SPATIAL = "Spatial"
+}
+
+# The datatype for which the indexing behavior is applied to.
+# 
+# + STRING - Represents a string data type
+# + NUMBER - Represents a numeric data type
+# + POINT - Represents a point data type
+# + POLYGON - Represents a polygon data type
+# + LINESTRING - Represents a line string data type
+public enum IndexDataType {
+    STRING = "String",
+    NUMBER = "Number",
+    POINT = "Point",
+    POLYGON = "Polygon",
+    LINESTRING = "LineString"
+}
+
+# Type of operation that invokes the trigger.
+# 
+# + ALL - `All` where trigger fires in all create, replace and delete operations
+# + CREATE - `Create` trigger fires only in a create operations
+# + REPLACE - `Replace` trigger fires only in a replace operations
+# + DELETE - `Delete` trigger fires only in a delete operations
+public enum TriggerOperation {
+    ALL = "All",
+    CREATE = "Create",
+    REPLACE = "Replace",
+    DELETE = "Delete"
+}
+
+# When is the trigger is fired.
+# 
+# + PRE - `Pre` triggers fire before an operation 
+# + POST - `Post` triggers fires after an operation
+public enum TriggerType {
+    PRE = "Pre",
+    POST = "Post"
+}
+
+# The access mode for the resource.
+# + ALL_PERMISSION - `All` provides read, write, and delete access to a resource
+# + READ_PERMISSION - `Read` restricts the user to read access on the resource
+public enum PermisssionMode {
+    ALL_PERMISSION = "All",
+    READ_PERMISSION = "Read"
+}
+
+# The specific version for a given offer.
+# 
+# + PRE_DEFINED - `V1` represents pre-defined throughput levels
+# + USER_DEFINED - `V2` represents user-defined throughput levels
+public enum OfferVersion {
+    PRE_DEFINED = "V1",
+    USER_DEFINED = "V2"
+}
+
+# The performance levels for a specific throughput level.
+# 
+# + LEVEL_S1 - `S1` represents performance level for pre-defined throughput level 
```

Comment:
IMO, we don't need to start with `S1 represents` etc as it rendered correctly as a chart in API docs (See example in [1] ) So can we simply say `Performance level for pre-defined throughput level`

Same comment applies for all the other places as well.

[1] https://ballerina.io/learn/api-docs/ballerina/#/ballerina/time/1.0.7/time/enums/DateTimeFormat

Reply:
Resolved this